contracts/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply from: '../buildsystem/android_library.gradle'
+apply from: '../buildsystem/coverageReport.gradle'
```

Comment:
This module just contains generated files which won't be tested.

Reply:
Fixed in 0a133dbfa206012fcdaa9a90c4a5c8b51a9d7331